src/main/java/homework_3/library/LibraryCard.java

```diff
@@ -27,7 +27,8 @@
     private final Calendar yearOfPublishing;
     private final int numberOfPages;
```

Comment:
Your class is 100% mutable

Reply:
A, Calendar is a reference object, that's why I have to return not it but also it's copy as Author?

Reply:
We care about reference type, mutable fields (you have 2 - Author and Calendar)

Reply:
Still not correct) Repeat the topic when you have time

Reply:
I will look at smarter students, there is something I do not understand but I want very much.

Reply:
Ok, I understood, we needn't setters at all and we have to override some getters for reference fields.